libs/core/src/components/pds-button/docs/pds-button.mdx

```diff
@@ -14,8 +14,6 @@ A button is an interactive element that can be clicked to perform an action or s
 - Progressing or regressing a user through a step in a flow (e.g. modals).
 
 ### When not to use
-
-- Directing users to a new page or different part within the same page. Instead, use [Link](/docs/components-link--external).
```

Comment:
I believe this should still stay. Adding a href in this case doesn't make it a link, it has a different purpose. 

A potential replacement of when to use CTA
```
When designing CTA buttons, choosing the correct element based on functionality is important. Use a <button> when the CTA triggers an action, such as submitting a form, opening a modal, or executing JavaScript. If the CTA is meant to navigate users to another page, section, or external link, an <a href=""> should be used instead, ensuring it is styled appropriately to look like a button.```

Reply:
Not sure I understand:

> "Adding a href in this case doesn't make it a link..."

Please elaborate, since that's exactly what it does. I am prob misunderstanding.